runtime/tests/interpreter/import_test.go

```diff
@@ -115,3 +118,144 @@ func TestInterpretVirtualImport(t *testing.T) {
 		value,
 	)
 }
+
+func TestInterpretImportMultipleProgramsFromLocation(t *testing.T) {
```

Comment:
This is the most important new test: It demonstrates how two declarations (`a` and `b`) can be imported from the same location (address location `0x1`), are resolved to two locations, which are backed by two programs.